src/data/finset/pointwise.lean

```diff
@@ -435,6 +470,36 @@ localized "attribute [instance] finset.mul_one_class finset.add_zero_class finse
   finset.div_inv_monoid finset.sub_neg_add_monoid finset.div_inv_monoid'"
   in pointwise
 
+section group
+variables [group α] {s t : finset α}
+
+@[to_additive] protected lemma mul_eq_one_iff : s * t = 1 ↔ ∃ a b, s = {a} ∧ t = {b} ∧ a * b = 1 :=
+by simp_rw [←coe_inj, coe_mul, coe_one, set.mul_eq_one_iff, coe_singleton]
+
+--TODO: Generalize to division monoids (#13860)
```

Comment:
#13860 is now merged

Reply:
I need a fair bit more. At least #14000.